integration-tests/tests/specs/features/revise-sequence.spec.ts

```diff
@@ -40,12 +40,14 @@ sequenceTest(
         await page.getByRole('link', { name: 'Revise this sequence' }).click({ timeout: 15000 });
         await expect(page.getByRole('heading', { name: 'Create new revision from' })).toBeVisible();
 
-        await page.getByTestId('discard_L_segment_file').click();
-        await page.getByTestId('discard_S_segment_file').click();
-        await page.getByTestId('S_segment_file').setInputFiles({
+        await page.getByTestId(/^discard.*L\)_segment_file$/).click();
+        await page.getByTestId(/^discard.*S\)_segment_file$/).click();
```

Comment:
Why the wildcard here? Don't we know the exact testid?

Reply:
Resolved in #5617